src/labs_examples/conditions_loops/labs/Exercise_03.java

```diff
@@ -7,7 +7,59 @@
  *      if the number from the user is 1, 2,... 12, or other respectively. Use a "switch" statement.
  *
  */
+import sun.lwawt.macosx.CSystemTray;
 
+import java.util.Scanner;
 public class Exercise_03 {
 
+
+    public static void main(String[] args) {
+        Scanner months = new Scanner(System.in);
+
+        System.out.println("Choose a number 1-12: Each number represents a month 1 being january: ");
+
+        int pOutput = months.nextInt();
```

Comment:
I would suggest using more telling variable names. 
I can't decifwr the `pOutput` name. As an example you could use `monthNumber` or `monthInput`.

Reply:
thats is a very good advice. I only put it that way because it was a small exercise but if i dont do it right I will start to build bad habits. Ive changed and pushed it here im not sure why its not showing up. but thank you